CHANGELOG.md

```diff
@@ -93,6 +93,7 @@
 
 - Change the default of `lua_ssl_trusted_certificate` to `system`
   [#8602](https://github.com/Kong/kong/pull/8602) to automatically load trusted CA list from system CA store.
+- `data_plane_config_cache_mode` and `data_plane_config_cache_path` were removed [#8704](https://github.com/Kong/kong/pull/8704).
```

Comment:
The data plane config cache mechanism and its related configuration options have been removed (`data_plane_config_cache_mode` and `data_plane_config_cache_path`) The config cache is now automatically persisted on disk with LMDB. With this change,  [#8704](https://github.com/Kong/kong/pull/8704).